packages/modules/data-widgets/src/themesource/datawidgets/web/datagrid.scss

```diff
@@ -329,6 +329,11 @@ $brand-primary: #264ae5 !default;
     -moz-box-shadow: 0 2px 20px 1px rgba(32, 43, 54, 0.08);
     box-shadow: 0 2px 20px 1px rgba(32, 43, 54, 0.08);
 
+    &.overflow {
+        height: 250px;
```

Comment:
is this just a number or we have a calculation for that?

Reply:
It's just a number, unsure how many elements should be visible in this case, how should I approach this?

Reply:
that's an interesting question let's see what @leonardomendix and @iobuhov thinks about that

Reply:
Let it be just as it is, this number is fine for now we can adjust it later.